xml/docker_concept.xml

```diff
@@ -0,0 +1,498 @@
+  <title>Limitations of &deng;</title>
+  <para>
+   On the other hand, &deng; has the following limitations:
+  </para>
+  <itemizedlist mark="bullet" spacing="normal">
+   <listitem>
+    <para>
+     Containers run inside the host system's kernel and cannot use a different
+     kernel.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Only allows Linux <emphasis>guest</emphasis> operating systems.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     &deng; is not a full virtualization stack like &xen;, &kvm;, or &libvirt;.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Security depends on the host system. Refer to the
+     <link xlink:href="http://docs.docker.com/articles/security/">official
+      security documentation</link> for more details.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="sec.docker.concept.images-applications">
+  <title>&docker; Images of Applications</title>
+  <para>
+  &deng; is a technology that can help minimize resources used to run or
+  build applications. There are several types of applications that are
+  suitable to run inside a &docker; container like daemons, Web pages or
+  applications that expose ports for communication. You can use &deng; to
+  automate building and deployment processes by adding the build process
+  into a &docker; image, then building the image and then running containers
+  based on that image.
+ </para>
+ <sect2 xml:id="sec.docker.concept.images-applications.run-inside">
+  <title>Advantages of Running Applications Inside &docker; Containers</title>
+  <para>
+   Running an application inside a &docker; container has the
+   following advantages:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     You can minimize the runtime environment of the application as you can add
+     to the &docker; image of the application just the required processes and
+     applications.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     The image with your application is portable across machines also with
+     different Linux host systems.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     You can share the image of your application by using a repository.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     You can use different versions of required packages in the container than
+     the host system uses without having problems with dependencies.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     You can run several instances of the same application that are completely
+     independent from each other.
+    </para>
+   </listitem>
+  </itemizedlist>
+  </sect2>
+  <sect2 xml:id="sec.docker.concept.images-applications.build">
+   <title>Advantages of &deng; to Build Applications</title>
+   <para>
+    Using &deng; to build applications has the following advantages:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      You can prepare a complete building image.
```

Comment:
I have trouble understanding "complete building image":
- "complete build image"? 
- completely building image? (but that would not make any sense)